cilium/cmd/helpers.go

```diff
@@ -377,11 +388,43 @@ func dumpConfig(Opts map[string]string) {
 		value = Opts[k]
 		if enabled, err := option.NormalizeBool(value); err != nil {
 			// If it cannot be parsed as a bool, just format the value.
-			fmt.Printf("%-24s %s\n", k, value)
+			if indented {
+				fmt.Printf("\t%-26s: %s\n", k, value)
+			} else {
+				fmt.Printf("%-34s: %s\n", k, value)
+			}
```

Comment:
The `if indented {...} else {...}` could be moved to line 388, to simplify the code here. Basically, you'd have only:
```go
formatStr := "%-34s: %s\n"
if indented {
    formatStr = "\t%-26s: %s\n"
}
```